podcasts/PlayerChapterCell.swift

```diff
@@ -111,6 +112,16 @@ class PlayerChapterCell: UITableViewCell {
     @IBAction func toggleChapterTapped(_ sender: Any) {
         chapter?.shouldPlay.toggle()
         toggleChapterButton.currentlyOn.toggle()
+
+        if let currentEpisode = PlaybackManager.shared.currentEpisode(), let index = chapter?.index {
+            if chapter?.shouldPlay == true {
+                currentEpisode.select(chapterIndex: index)
+            } else {
+                currentEpisode.deselect(chapterIndex: index)
+            }
+
+            DataManager.sharedManager.save(episode: currentEpisode)
```

Comment:
I thought about moving this call to `BaseEpisode`. However, the UI design will change and I believe the logic on where we save users' selection will probably change, so I decided to just leave that here for now.